.github/workflows/push.yml

```diff
@@ -0,0 +1,174 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Intel Corporation
+
+name: Tag/Publish Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
```

Comment:
this looks great - I presume you're reusing from sdran-helm-charts

Reply:
No, the `sdran-helm-charts` does not have GHAs. I am reusing the GHAs I created for the `sdcore-helm-charts`